packages/mjml-validator/src/dependencies.js

```diff
@@ -1,14 +1,26 @@
 export const assignDependencies = (target, ...sources) => {
+  if (!sources) {
```

Comment:
Better check length. `sources` is always an array

Reply:
Done.